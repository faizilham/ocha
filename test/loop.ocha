let total = 0;
let i = 0;

while (i < 100) {
    i = i + 1;
    total = total + i;
}

print(total);                   // 5050

let str = "";
i = 0;
let j;

while (i < 2) {
    j = 0;
    str = str + "a";
    while (j < 100) {
        str = str + "b";
        j = j + 1;
        if (j == 2) {
            break;
        }
    }
    i = i + 1;
}

print(str);                     // abbabb
