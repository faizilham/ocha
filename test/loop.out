5050
abbabb
5050
